examples/Euler_examples.jl

```diff
@@ -0,0 +1,85 @@
+using OrdinaryDiffEq
+using Trixi
+using Plots
+
+###############################################################################
+# setting up source term
+#function of form S(Q) where Q = (ρ, ρux, ρuy, ρE)
+@inline function source_test(u, x, t, equations)
+    #in 1D for now so we are leaving out y
+    ϕ = tan(x[1])^(-1) #arctan(y/x)
+    ρ = u[1] #density
+    G = 6.67408 * 10^(-11) #gravitational constant
+    mₛ = 4 * 1.989 * 10^(30) #mass of central object
+    ux = u[2] #velocity of x
+    uy = u[3] #velocity of y
+    r = sqrt(x[1]^2) #√(x² + y²)
+
+    du1 = -cos(ϕ)*ρ*(G*mₛ/(r^2))
+    du2 = -sin(ϕ)*ρ*(G*mₛ/(r^2))
+    du3 = -(ux*cos(ϕ) + uy*sin(ϕ))*ρ*(G*mₛ/(r^2))
+    return SVector(du1, du2, du3)
```

Comment:
Careful! Dropped the wrong dimension. We want to get rid of $u_y$, so the terms this function should return are like
```julia
SVector(0, d1, d3)
```

(In Julia it's just `atan` :p)